pkg/controllers/user/systemimage/systemimage.go

```diff
@@ -0,0 +1,187 @@
+package systemimage
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/rancher/norman/types/convert"
+	"github.com/rancher/rancher/pkg/image"
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/types/apis/apps/v1beta2"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+const (
+	systemImageVersionAnn = "field.cattle.io/systemImageVersion"
+	systemProjectAnn      = "management.cattle.io/system-namespace"
```

Comment:
this annotation is used in a couple of other places in the code. Lets again define the constant in some common place